packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -408,27 +408,6 @@ describe('packages/date-picker', () => {
         expect(menuContainerEl).toBeInTheDocument();
       });
 
-      test('appends to the end of the DOM', async () => {
```

Comment:
was going to comment that maybe we want to test something else instead, but that would just be re-testing portal functionality. I think this is correct, and we don't want to re-test that here.

Reply:
OTOH, it may be useful to test that a component extends Popover correctly (e.g. a custom expect? `expect(X).toExtendPopover()`)?